src/extension/infer.rs

```diff
@@ -652,7 +652,8 @@ impl UnificationContext {
             new_variables = new_variables
                 .into_iter()
                 .filter(|m| seen.insert(*m))
-                .flat_map(|m| self.get_constraints(&m).unwrap())
+                .flat_map(|m| self.get_constraints(&m))
```

Comment:
@croyzor this fixes a crash (unwrap of None) here, for a meta that has a solution but no constraints. Does that look ok?

Reply:
I think so 👍